components/profile_popover/profile_popover.jsx

```diff
@@ -323,6 +330,24 @@ class ProfilePopover extends React.Component {
             );
         }
 
+        if (this.props.enableTimezone) {
+            const currentTimezone = getCurrentTimezone(this.props.user.timezone);
+            if (currentTimezone) {
+                const useMilitaryTime = PreferenceStore.getBool(Constants.Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time');
+                const date = new Date();
+                dataContent.push(
+                    <span>
```

Comment:
Would it make sense to factor out a `<DateTime>` component that connects the `useMilitaryTime` and `timezone` preferences vs. having to pass them into every component that needs to render a time?

Reply:
that would be a nice refactor 👍